hybridse/include/node/node_manager.h

```diff
@@ -309,6 +309,7 @@ class NodeManager {
     CreatePlanNode *MakeCreateTablePlanNode(
         const std::string& db_name,
         const std::string &table_name, int replica_num, int partition_num,
+        const std::string& storage_mode,
```

Comment:
can this be a enum ?

Reply:
Actually there is an `enum StorageMode` in `src/proto/common.proto`. But I'm not sure it is good to include header `src/proto/common.pb.h` in `hybridse` codebase. 

Is it ok to add `${PROJECT_SOURCE_DIR}/../src/proto` or `${PROJECT_SOURCE_DIR}/..`  to `include_directories` in `hybridse/CMakeLists.txt`?

Reply:
better not, currently hybridse is a dependency to src. Maybe we can define separate enum, save space at least